### ISSUE TITLE
Low test coverage!

### ISSUE THREAD

**Issue by amihaiemil** [yegor256/0rsk#32]
```
Cause: Test coverage is under 40% (39% at the moment of writing).
Risk: Bugs may be easily introduced
Effect: Users will not be happy
```

Did I get this right? :)

**Comment by yegor256** [comment]
@amihaiemil perfectly :)

**Comment by yegor256** [comment]
@amihaiemil but it's not true. The reporting is broken. For example: https://codecov.io/gh/yegor256/0rsk/src/master/objects/tasks.rb This page says that `tasks.rb` is not covered, but we have `test_tasks.rb`, which covers this class perfectly.

**Comment by pnatashap** [comment]
Looks like codecov failed: 
`/home/runner/work/0rsk/0rsk/objects/plans.rb:25: warning: /home/runner/work/0rsk/0rsk/objects/plans.rb:25: warning: loading in progress, circular require considered harmful - /home/runner/work/0rsk/0rsk/objects/tasks.rb
	from /opt/hostedtoolcache/Ruby/2.7.8/x64/lib/ruby/gems/2.7.0/gems/rake-13.1.0/lib/rake/rake_test_loader.rb:6:in  `<main>'
	from /opt/hostedtoolcache/Ruby/2.7.8/x64/lib/ruby/gems/2.7.0/gems/rake-13.1.0/lib/rake/rake_test_loader.rb:6:in  `select'
	from /opt/hostedtoolcache/Ruby/2.7.8/x64/lib/ruby/gems/2.7.0/gems/rake-13.1.0/lib/rake/rake_test_loader.rb:21:in  `block in <main>'
	from /opt/hostedtoolcache/Ruby/2.7.8/x64/lib/ruby/gems/2.7.0/gems/rake-13.1.0/lib/rake/rake_test_loader.rb:21:in  `require'
	from /home/runner/work/0rsk/0rsk/test/test_0rsk.rb:26:in  `<top (required)>'
	from /home/runner/work/0rsk/0rsk/test/test_0rsk.rb:26:in  `require_relative'
	from /home/runner/work/0rsk/0rsk/0rsk.rb:299:in  `<top (required)>'
	from /home/runner/work/0rsk/0rsk/0rsk.rb:299:in  `require_relative'
	from /home/runner/work/0rsk/0rsk/front/front_tasks.rb:23:in  `<top (required)>'
	from /home/runner/work/0rsk/0rsk/front/front_tasks.rb:23:in  `require_relative'
	from /home/runner/work/0rsk/0rsk/objects/tasks.rb:24:in  `<top (required)>'
	from /home/runner/work/0rsk/0rsk/objects/tasks.rb:24:in  `require_relative'
	from /home/runner/work/0rsk/0rsk/objects/plans.rb:25:in  `<top (required)>'
	from /home/runner/work/0rsk/0rsk/objects/plans.rb:25:in  `require_relative'

Coverage report generated for Unit Tests to /home/runner/work/0rsk/0rsk/coverage/coverage.xml. 291 / 744 LOC (39.11%) covered`

**Comment by yegor256** [comment]
@pnatashap can we fix this?

**Comment by pnatashap** [comment]
Yes, I fix that error, but in addition codecov report is generated before test run, so no logic is covered and new test does't affect on it. Looking into it, very strange

**Comment by pnatashap** [comment]
Some error are fixed in PR, but coverage is still low. 
`Coverage report generated for Unit Tests to /home/runner/work/0rsk/0rsk/coverage/coverage.xml. 285 / 735 LOC (38.78%) covered
Started with run options --seed 63982`
The same issue is in 0pdd, but works good in netbout (with similar code). The only diff that I can see - in netbout codecov report is generated after tests run.

**Comment by artur-intech** [comment]
@yegor256 Is this still valid?
